atomic_reactor/util.py

```diff
@@ -1252,13 +1252,13 @@ def validate_with_schema(data: dict, schema: str, package: str = "atomic_reactor
     osbs_yaml.validate_with_schema(data, schema)
 
 
-def allow_repo_dir_in_dockerignore(build_path):
+def allow_path_in_dockerignore(build_path, allow_path):
     docker_ignore = os.path.join(str(build_path), DOCKERIGNORE)
 
     if os.path.isfile(docker_ignore):
         with open(docker_ignore, "a") as f:
-            f.write("\n!%s\n" % RELATIVE_REPOS_PATH)
-        logger.debug("Allowing %s in %s", RELATIVE_REPOS_PATH, DOCKERIGNORE)
+            f.write("\n!%s\n" % allow_path)
```

Comment:
didn't we agreed on new style format string or F-strings?
```suggestion
            f.write(f"\n!{allow_path}\n")
```

Reply:
ok